diff --git a/postinstall.mjs b/postinstall.mjs
index 59dce3578c786444b2fa1f6534e93d94b8863405..4685d6891d5b05501b67ae1786905dc9269d4417 100644
--- a/postinstall.mjs
+++ b/postinstall.mjs
@@ -11,7 +11,7 @@ const arch = process.arch === "arm64" ? "arm64" : "x64";
 const { version } = pkg;
 const binUrl = `https://github.com/aklinker1/buildc/releases/download/v${version}/buildc-${os}-${arch}${ext}`;
 
-const downloadTo = path.resolve("buildc");
+const downloadTo = path.resolve("buildc" + ext);
 
 console.log("Downloading binary:", binUrl);
 console.log("To:", downloadTo);
