diff --git a/dist/index.js b/dist/index.js
index e2626a009efe1ee3cfa6c96fe712341d3501ba1c..c33c118d149271c3571e50eb2fdc91ccae801acb 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3756,6 +3756,7 @@ async function resolveTarball(pm, p3, pJson) {
   if (pm === "bun") {
     const tgzFiles = fsSync.readdirSync(p3).filter((file) => file.endsWith(".tgz"));
     const bunFilename = stdout2.trim();
+    console.log({ tgzFiles, bunFilename })
     if (bunFilename) {
       filename = bunFilename;
     }
@@ -3765,6 +3766,7 @@ ${stdout2}`);
       `[bun pack] expected filename: ${filename}; tgz files: ${tgzFiles.join(", ") || "(none)"}`
     );
   }
+  console.log({ filename })
   const shasum = createHash("sha1").update(await fs.readFile(path.resolve(p3, filename))).digest("hex");
   return { filename, shasum };
 }
