diff --git a/dist/index.js b/dist/index.js
index e2626a009efe1ee3cfa6c96fe712341d3501ba1c..f3b45e7cad0bfbbb877eac6a22fdf48051127d11 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3743,7 +3743,7 @@ async function resolveTarball(pm, p3, pJson) {
   if (pm === "yarn") {
     cmd += ` --filename ${filename}`;
   } else if (pm === "bun") {
-    cmd = `bun pm pack --quiet --filename ${filename}`;
+    cmd = `bun pm pack --quiet --filename ${p3}${filename}`;
   }
   const { stdout: stdout2 } = await ezSpawn.async(cmd, {
     stdio: "overlapped",
